client/src/components/vote/vote-single/vote-single.html

```diff
@@ -138,33 +167,43 @@ <h3>
           </md-icon>
           <div class="md-list-item-text">
             <h3>
-              {{ $ctrl.rating.current.entry.resolution / 1000000 | number:2 }} Mpix
+              {{ $ctrl.rating.current.entry.resolution / 1000000 | number: 2 }}
+              Mpix
             </h3>
             <p>
               {{ $ctrl.rating.current.entry.width }} ×
               {{ $ctrl.rating.current.entry.height }}
             </p>
           </div>
         </md-list-item>
-        <md-list-item class="md-2-line"
-            ng-show="$ctrl.rating.current.history">
+        <md-list-item class="md-2-line" ng-show="$ctrl.rating.current.history">
           <md-icon>
             history
             <md-tooltip>File history</md-tooltip>
           </md-icon>
           <div class="md-list-item-text">
             <h3>
               {{ $ctrl.rating.current.history.length }}
-              version<span ng-if="$ctrl.rating.current.history.length > 1">s</span>
+              version<span ng-if="$ctrl.rating.current.history.length > 1"
+                >s</span
+              >
             </h3>
             <p>
               last one at
-              {{ $ctrl.rating.current.history[0].timestamp | date:'d MMM yyyy' }}
+              {{
+                $ctrl.rating.current.history[0].timestamp | date: "d MMM yyyy"
+              }}
             </p>
           </div>
-          <md-button class="md-secondary md-icon-button"
-              ng-href="{{ '//commons.wikimedia.org/wiki/File:' + $ctrl.rating.current.entry.name + '#filehistory' }}"
-              target="_blank">
+          <md-button
+            class="md-secondary md-icon-button"
+            ng-href="{{
+              '//commons.wikimedia.org/wiki/File:' +
+                $ctrl.rating.current.entry.name +
```

Comment:
is this going to need escaping or is `entry.name` already url escaped?

Reply:
I think we're getting `entry.name` straight from mediawiki, so it should be URL friendly. 

We should have a test for this, though—`$ctrl.rating.current.entry.name` makes an appearance above, and there is `encodeURIComponent(image.entry.name)` elsewhere...